packages/theming/atlas/src/theme/native/custom-variables.ts

```diff
@@ -582,3 +582,105 @@ export const slider: VariablesSlider = {
         backgroundColor: background.gray
     }
 };
+
+// == Variables to exclude files
```

Comment:
These variables should also be included in the themesource folder to have defaults. For web, @cjfhodges wanted them in a separate file `themesource/web/core_variables.scss`, so I think you need to do the same in `themesource/native/coreVariables.ts`

Reply:
I dont understand the benefit of having defaults for these variables on our `themesource` folder. This is purely created for the customers and we won't be flipping these variables by ourselves.

Reply:
In the case that users strip their custom variables file, we ensure we always have a value for the variable in Atlas.

Reply:
Something like this 
$exclude-button-helpers: false;

@if variable-exists(exclude-button-helpers)
        @if $exclude-button-helpers == false{
                @import "core/helpers/buttons";
 }
}